### PR TITLE
hollywood: new, 1.21-5 

### PR DESCRIPTION
**Pull request by PegionFish**
Topic Description
-----------------
Launch Byobu
Open a random number of splits, random sizes
In each split, run a noisy text app

Rules:
 - Must work as a non-root user
 - Must display information indefinitely (a la "watch", or cmatrix)
   + Can use a while/true + a timeout
 - Must not OOM a system
 - Must not over load a system
 - Must not be too egregious with I/O
 - Must not require outbound internet access

*copied from https://github.com/dustinkirkland/hollywood/blob/master/README *

Package(s) Affected
-------------------
hollywood

Security Update?
----------------

No

Architectural Progress
----------------------

- [ ] AMD64 `amd64`   

    

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [ ] AMD64 `amd64`   

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

### CODE REVIEW

File: extra-misc/hollywood/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PKGNAME=hollywood
+PKGSEC=extra
```
Comment (MingcongBai) [inline extra-misc/hollywood/autobuild/defines]: `extra` is not a valid section. Please refer to `sets/section` for a list of valid sections. This probably goes into `utils`.

File: extra-misc/hollywood/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+PKGDEP="apg bmon byobu cmatrix coreutils gawk htop man-db mlocate moreutils mplayer openssh speedometer tree util-linux"
```
Comment (MingcongBai) [inline extra-misc/hollywood/autobuild/defines]: I doubt that all of these are actual dependencies, especially `byobu`, `mplayer`, ... I think these are supposed to serve as frontends? Please only include all those required for `hollywood` to run as dependencies, and all others should go to `PKGSUG` as optional dependencies.

File: extra-misc/hollywood/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+PKGDES="Fill your console with Hollywood melodrama technobabble."
```
Comment (MingcongBai) [inline extra-misc/hollywood/autobuild/defines]: Drop the period point at the end for `PKGDES`.

File: extra-misc/hollywood/spec [hunk 4]
```diff
@@ -0,0 +1,3 @@
+VER=1.21
+SRCS="https://launchpad.net/hollywood/trunk/1.21/+download/hollywood_1.21.orig.tar.gz"
```
Comment (MingcongBai) [inline extra-misc/hollywood/spec]: Hardcoded version here, please substitute them with `$VER`.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-misc/hollywood/autobuild/defines (hunk 1 above)
Comment (PegionFish) [reply extra-misc/hollywood/autobuild/defines]: I'm not sure where this should go & extra seems to be a good target. WIP

File: extra-misc/hollywood/autobuild/defines (hunk 2 above)
Comment (PegionFish) [reply extra-misc/hollywood/autobuild/defines]: These packages are required to run as dependencies as the README shows.

Comment (PegionFish) [reply extra-misc/hollywood/autobuild/defines]: WIP

File: extra-misc/hollywood/spec (hunk 4 above)
Comment (PegionFish) [reply extra-misc/hollywood/spec]: fixed

File: extra-misc/hollywood/autobuild/defines (hunk 1 above)
Comment (MingcongBai) [reply extra-misc/hollywood/autobuild/defines]: Extra is not even a valid section. Try `utils`.

File: extra-misc/hollywood/autobuild/defines (hunk 2 above)
Comment (MingcongBai) [reply extra-misc/hollywood/autobuild/defines]: I don't know where you got this from, nothing like that is mentioned in the README, unless you meant "sample Apps."

**Pull request closed without merging**